app/models/post.rb

```diff
@@ -0,0 +1,14 @@
+class Post < ActiveRecord::Base
+  has_attached_file :image, :styles => { :medium => "300x300>", :thumb => "200x200>" }, :default_url => "/images/:style/missing.png"
+  validates_attachment_content_type :image, :content_type => /\Aimage\/.*\Z/
+
+  def build_comment(attributes = {}, user)
```

Comment:
Style/OptionalArguments: Optional arguments should appear at the end of the argument list. (https://github.com/bbatsov/ruby-style-guide#optional-arguments)